flink-tests/src/test/java/org/apache/flink/runtime/operators/lifecycle/TestJobExecutor.java

```diff
@@ -77,28 +77,28 @@ private TestJobExecutor(
     public static TestJobExecutor execute(
             TestJobWithDescription testJob, MiniClusterWithClientResource miniClusterResource)
             throws Exception {
-        LOG.debug("submitGraph: {}", testJob.jobGraph);
+        LOG.info("submitGraph: {}", testJob.jobGraph);
         JobID job = miniClusterResource.getClusterClient().submitJob(testJob.jobGraph).get();
         waitForAllTaskRunning(miniClusterResource.getMiniCluster(), job, false);
         return new TestJobExecutor(testJob, job, miniClusterResource);
     }
 
     public TestJobExecutor waitForAllRunning() throws Exception {
-        LOG.debug("waitForAllRunning in {}", jobID);
+        LOG.info("waitForAllRunning in {}", jobID);
```

Comment:
I think it should be actually info because this is some test execution phase.
Besides that, changing in tools/ci/log4j.properties would only help when running on CI, not locally for example.

So I'd rather keep it `info` in code.